tests/__init__.py

```diff
@@ -30,12 +30,11 @@
 Standard set of databases to run tests against
 """
 
-LOCATION = "/var/tmp/mdapi-tests/"
+LOCATION = "/var/tmp/mdapi-tests/"  # noqa : S108
 BRCHNAME = "rawhide"
 PROBEURL = {
-    "koji": "https://kojipkgs.fedoraproject.org/repos/%s/latest/x86_64/repodata/" % BRCHNAME,
-    "rawhide": "https://dl.fedoraproject.org/pub/fedora/linux/development/%s/Everything/x86_64/os/repodata/"  # noqa
-    % BRCHNAME,
+    "koji": f"https://kojipkgs.fedoraproject.org/repos/{BRCHNAME}/latest/x86_64/repodata/",
+    "rawhide": f"https://dl.fedoraproject.org/pub/fedora/linux/development/{BRCHNAME}/Everything/x86_64/os/repodata/",
```

Comment:
As discussed, this can break.

Reply:
Not sure if you mean that the "line" can break (vide `E501`) or if the downloading part, the failure of which can break the tests? Assuming it is the latter, that is a scenario I am totally fine with as even the downloading part needs testing here.